showup/models.py

```diff
@@ -2,6 +2,16 @@
 from django.contrib.auth.models import AbstractUser
 
 
+class Genre(models.Model):
+    genre = models.TextField()
+
+    class Meta:
+        constraints = [models.UniqueConstraint(fields=["genre"], name="Genre_unique")]
```

Comment:
what does this code do

Reply:
It's a unique constraint so that we won't have any genre more than once

Reply:
ok thx